code/game/objects/items/weapons/dice.dm

```diff
@@ -339,6 +354,37 @@
 /obj/item/dice/attack_self(mob/user)
 	diceroll(user)
 
+/obj/item/dice/attack(mob/living/M, mob/user)
+	if(M == user)
+		var/mob/living/carbon/human/H = user
+		if(!H.check_has_mouth())
+			to_chat(user, "<span class='warning'>You do not have a mouth!</span>")
+			return
+		if(times_eaten < max_bites)
+			times_eaten++
+			to_chat(user, "<span class='notice'>You swallow the dice. Delicious!</span>")
+			diceroll(H)
+			to_chat(user, "<span class='danger'>You swallowed a [result]!</span>")
+			H.AdjustLoseBreath(result SECONDS)
```

Comment:
Correct me if I'm wrong, but this could quite easily be lethal when stacked (or with a d100), no? As much as I love the random damage quirk, I'm not sure we want something so common to be a viable murder weapon (not mentioning other side effects loss of breath has, like ignoring gases)